webapp/packages/plugin-authentication-administration/src/Administration/Users/Teams/TeamsForm/Options/TeamOptionsFormPart.ts

```diff
@@ -116,6 +116,7 @@ export class TeamOptionsFormPart extends FormPart<ITeamOptionsState, ITeamFormSt
       const team = await this.teamResource.createTeam(teamInfo);
       await this.teamsMetaParametersResource.setMetaParameters(this.state.teamId, this.state.metaParameters);
 
+      this.formState.setMode(FormMode.Edit);
       this.formState.setState({
         teamId: team.teamId,
       });
```

Comment:
You should change form mode right after `await this.teamResource.createTeam(teamInfo);`
So in case `await this.teamsMetaParametersResource.setMetaParameters(this.state.teamId, this.state.metaParameters);` will throw an exception, it will be appropriately handled. Right now form will think that is still creating user and will execute `await this.teamResource.createTeam(teamInfo);` again